packages/headless/src/helper/serviceResponse/index.ts

```diff
@@ -10,4 +11,9 @@ export class ServiceResponse implements IServiceResponse {
     errorResponse(error: string, errors: DescriptiveError | null, code: number = HttpStatus.NOT_FOUND): ServiceErrorResponse {
         return ({ error, errors, code });
     }
+
+    graphqlResponse(res: any): object {
```

Comment:
would you please change the type of "res"? we can user proper type here. also template might be used for return type.